mesheryctl/internal/cli/root/system/reset_test.go

```diff
@@ -0,0 +1,87 @@
+package system
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	log "github.com/sirupsen/logrus"
+)
+
+//var update1 = flag.Bool("update", true, "update golden files")
+
+// This is a Unit test
+func TestResetCmd(t *testing.T) {
+	_, filename, _, ok := runtime.Caller(0)
+	if !ok {
+		t.Fatal("Not able to get current working directory")
+	}
+	currDir := filepath.Dir(filename)
+	utils.SetupCustomContextEnv(t, currDir+"/testdata/reset/TestResetContexts.yaml")
+
+	tests := []utils.CmdTestInput{
+		//Test for platform docker
+		{
+			Name:             "Reset the meshery config file with docker platform",
+			Args:             []string{"reset", "-y", "-c", "local2"},
+			ExpectedResponse: "reset_docker.output.golden",
+		},
+		//Test for platform kubernetes
+		{
+			Name:             "Reset the meshery config file with kubernetes platform",
+			Args:             []string{"reset", "-y", "-c", "local"},
+			ExpectedResponse: "reset_kubernetes.output.golden",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.Name, func(t *testing.T) {
+			// setting up log to grab logs
+			b := utils.SetupLogrusGrabTesting(t)
+			SystemCmd.SetOut(b)
+			SystemCmd.SetArgs(tt.Args)
+			err = SystemCmd.Execute()
+			if err != nil {
+				t.Error(err)
+			}
+			//Collecting the actual response
+			actualResponse := b.String()
+
+			testdataDir := filepath.Join(currDir, "testdata/reset/")
+			golden := utils.NewGoldenFile(t, tt.ExpectedResponse, testdataDir)
+
+			//if *update1 {
+			//	golden.Write(actualResponse)
+			//}
```

Comment:
why this has been commented out?

Reply:
I get this error if that line is uncommented.

<img width="1381" alt="Screenshot 2021-08-08 at 2 06 37 PM" src="https://user-images.githubusercontent.com/64970095/128626216-99e0e51c-2690-4f2c-b0d4-228ac903a7eb.png">

Reply:
It giving me an error that `flag redefined : update`